.github/workflows/ci-uxd.yml

```diff
@@ -0,0 +1,143 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Would these tests only run on main? I would add a v.** wildcard, release are/will-be done from branches like e.g. v3.3.0

Reply:
ah my bad, it should be v**, just looked up